deployment/ansible/roles/model-my-watershed.beaver/templates/spark-jobserver.conf.j2

```diff
@@ -1,4 +0,0 @@
-[/var/log/upstart/spark-jobserver.log]
-multiline_regex_before = ^\s+
-type: spark-jobserver
-tags: spark-jobserver,beaver
```

Comment:
Wondering know if the akka log should live in the `upstart` dir? If so, I can also create a new file for this log.